components/CheckInOrOut.tsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import {
+    Box,
+    Button,
+    Typography,
+    Modal,
+    TextField,
+    ThemeProvider,
+  } from '@mui/material';
+  import CloseIcon from '@mui/icons-material/Close';
+  import theme from './Theme';
+
+const style = {
+    position: 'absolute' as 'absolute',
+    top: '50%',
+    left: '50%',
+    transform: 'translate(-50%, -50%)',
+    width: 350,
+    bgcolor: 'background.paper',
+    boxShadow: 24,
+    padding: '32px 32px 12px 32px',
+};
```

Comment:
We're gonna want to clean this up with the rest of the css styles, no change necessary for now though